s3-uploads.php

```diff
@@ -21,6 +21,13 @@ function s3_uploads_init() {
 		return;
 	}
 
+	// Make sure the plugin is enabled when autoenable is on, or in a CLI system (so the migrate command works)
+	if ( ( ! defined( 'WP_CLI') || ! WP_CLI ) &&                                      // If CLI is enabled, skip our other checks
```

Comment:
Can we late-enable the plugin when the command is run for the CLI? This definitely needs to be simplified down from a giant conditional to multiple returns if the checks don't change too, multiline conditionals are terribly hard to read.